### PR TITLE
feat(wallet): update homepage design

### LINKED ISSUE #81: Update homepage design
<details>
  <summary>Preview homepage design (mobile)</summary>
  
image

</details>

<details>
  <summary>Preview homepage design (desktop)</summary>
  
image

</details>

### PR DESCRIPTION
**Pull request by raducristianpopa**
### Context
- closes #81 

### Changes
Design changes for homepage
- [x] Mobile
- [x] Desktop

### CODE REVIEW

File: packages/wallet/src/components/AccountCard.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        <div className="mt-auto text-white">
+          <p className="overflow-hidden text-ellipsis whitespace-nowrap font-light">
+            {name}
+          </p>
+          <p className="text-2xl font-semibold tracking-tighter">{balance}</p>
+        </div>
+      </div>
+    </div>
+  )
+}
```
Comment (Tymmmy) [inline packages/wallet/src/components/AccountCard.tsx:25]:
```suggestion
       <div className="flex aspect-square flex-1 flex-col p-2">
         <div className="flex flex-1 text-2xl font-semibold">
           <span className="inline-flex h-8 w-14 items-center justify-center rounded-md bg-white mix-blend-screen">
             {asset}
           </span>
         </div>
         <div className="mt-auto text-white">
           <p className="overflow-hidden text-ellipsis whitespace-nowrap font-light">
             {name}
           </p>
           <p className="text-2xl font-semibold tracking-tighter">{balance}</p>
         </div>
       </div>

```

Comment (Tymmmy) [reply packages/wallet/src/components/AccountCard.tsx:25]: I would remove one layer of div here.

File: packages/wallet/src/pages/index.tsx [hunk 2]
```diff
@@ -1,48 +1,82 @@
...
+          </Link>
+          <Link className="group flex aspect-square flex-col items-center justify-center rounded-lg border border-green-5 bg-white shadow-md hover:border-green-6">
+            <Receive className="h-8 w-8" />
+            <span className="font-medium text-green-5 group-hover:text-green-6">
+              Receive
+            </span>
+          </Link>
+          <Link className="group flex aspect-square flex-col items-center justify-center rounded-lg border border-green-5 bg-white shadow-md hover:border-green-6">
+            <New className="h-8 w-8" />
+            <span className="font-medium text-green-5 group-hover:text-green-6">
```
Comment (Tymmmy) [inline packages/wallet/src/pages/index.tsx:52]:
```suggestion
            <span className="font-medium leading-3 text-green-5 group-hover:text-green-6 sm:leading-none">
```

Comment (raducristianpopa) [reply packages/wallet/src/pages/index.tsx:52]: By doing this, the icons would not be correctly aligned on mobile devices. 🤔

**Review by Tymmmy — APPROVED** [review]

**Review by Tymmmy — APPROVED** [review]

**Pull request merged**